apps/web/src/pages/TalentNominations/NominationGroup/Layout.tsx

```diff
@@ -0,0 +1,179 @@
+import { useIntl } from "react-intl";
+import { Outlet } from "react-router";
+import { useQuery } from "urql";
+
+import { ROLE_NAME } from "@gc-digital-talent/auth";
+import { FragmentType, getFragment, graphql } from "@gc-digital-talent/graphql";
+import { Pending, Sidebar, ThrowNotFound } from "@gc-digital-talent/ui";
+import { commonMessages, navigationMessages } from "@gc-digital-talent/i18n";
+
+import Hero from "~/components/Hero";
+import RequireAuth from "~/components/RequireAuth/RequireAuth";
+import SEO from "~/components/SEO/SEO";
+import useRequiredParams from "~/hooks/useRequiredParams";
+import { getFullNameLabel } from "~/utils/nameUtils";
+import useBreadcrumbs from "~/hooks/useBreadcrumbs";
+import useRoutes from "~/hooks/useRoutes";
+import pageTitles from "~/messages/pageTitles";
+
+import { RouteParams } from "./types";
+
+const TalentNominationGroupLayout_Fragment = graphql(/* GraphQL */ `
+  fragment TalentNominationGroupLayout on TalentNominationGroup {
+    talentNominationEvent {
+      id
+      name {
+        localized
+      }
+    }
+    nominee {
+      firstName
+      lastName
+    }
+  }
+`);
+
+interface LayoutProps {
+  query: FragmentType<typeof TalentNominationGroupLayout_Fragment>;
+}
+
+const Layout = ({ query }: LayoutProps) => {
+  const intl = useIntl();
+  const paths = useRoutes();
+  const { talentNominationGroupId } = useRequiredParams<RouteParams>(
+    "talentNominationGroupId",
+  );
+  const talentNominationGroup = getFragment(
+    TalentNominationGroupLayout_Fragment,
+    query,
+  );
+
+  const nomineeName = getFullNameLabel(
+    talentNominationGroup.nominee?.firstName,
+    talentNominationGroup.nominee?.lastName,
+    intl,
+  );
+
+  const description = intl.formatMessage({
+    defaultMessage:
+      "Review a nominee’s profile and approve or reject their nomination.",
+    id: "0m1utE",
+    description: "Description of a talent nomination group",
+  });
+
+  const crumbs = useBreadcrumbs({
+    crumbs: [
+      {
+        label: intl.formatMessage(navigationMessages.communityDashboard),
+        url: paths.communityDashboard(),
+      },
+      {
+        label: intl.formatMessage(pageTitles.talentManagement),
+        url: "#", // NOTE: Page doesn't exist yet
```

Comment:
Sorry, I know this wasn't part of the AC.  I'm worried about forgetting to come back to these later.  Could you please add them to useRoutes and fill it into this layout?  The URLs will be `${adminUrl}/talent-events` and `${adminUrl}/talent-events/${eventId}`.

Reply:
That is a good call, added in [add future pages to routes](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13079/commits/96aa2aac4151b19b8f4234de4cea7d1dca2d39a9)